<?xml version="1.0"?>
<launch>
  
   <!-- roslaunch robotican_armadillo armadillo.launch kinect2:=true world_name:="`rospack find robotican_common`/worlds/objects_on_table.world" gazebo:=true -->
   <!-- roslaunch robotican_armadillo armadillo.launch kinect2:=true  -->
  
  
  
  <node name="find_object_for_track_node" pkg="robotican_common" type="find_objects_node" output="screen">
    <param name="~object_id" value="1"/>
    <param name="~depth_topic1" value="/kinect2/qhd/points"/>
     <param name="~depth_topic2" value="/sr300/depth/points"/>    
    <rosparam file="$(find robotican_demos)/config/blue_object.yaml" command="load"/>
   </node> 
   
     
    <node name="objects_handler" pkg="robotican_common" type="objects_handler" output="screen">
       <param name="~db_path" value=""/>
   </node> 
    
    <node name="object_view" pkg="rqt_image_view" type="rqt_image_view" >
      <remap from="image" to="find_object_node/result" />
    </node>
  
   <node name="pan_tilt_object_tracking_node" pkg="robotican_common" type="pan_tilt_object_tracking_node" output="screen">
    <param name="~object_name" value="can"/>
   </node> 
    
   
</launch>

